client/src/components/ReviewsList.jsx

```diff
@@ -1,99 +1,115 @@
+                <div>
+                  <select
+                    className='languageDropDown'>
+                    <option
+                      defaultValue='english'>
+                      Language English ({getRandomInclusive(3000, 5500)})
+                    </option>
+                    <option
+                      value='secondLanguage'>
+                      {reviews[0].languages}
+                    </option>
+                    <option
+                      value='thirdLanguage'>
+                      {reviews[1].languages}
+                    </option>
+                  </select>
+                </div>
+              </div>
+            </div>
           </div>
         </div>
-        <div
-          className='searchBarContainer'>
-          <div
-            className='searchBar'>
-            <input
-              type='text'
-              defaultValue='Search within the reviews'
-            />
-          </div>
-          <div
-            className='searchButton'>
-            <button>
-              Go
-            </button>
-          </div>
-        </div>
-        <div
-          className='sortContainer'>
-          <div
-            className='sortBy'>
-            Sort by
-          </div>
-          <div
-            className='sortDropDown'>
-            <select>
-              <option
-                defaultValue='yelp'>
-                Yelp Sort
-              </option>
-              <option
-                value='newest'>
-                Newest First
-              </option>
-              <option
-                value='oldest'>
-                Oldest First
-              </option>
-              <option
-                value='highest'>
-                Highest Rated
-              </option>
-              <option
-                value='lowest'>
-                Lowest Rated
-              </option>
-            </select>
-          </div>
-        </div>
-        <div
-          className='languageContainer'>
-          <div
-            className='language'>
-            Language
-          </div>
-          <div
-            className='languageDropDown'>
-            <select>
-              <option
-                defaultValue='english'>
-                English (5007)
-              </option>
-              <option
-                value='japanese'>
-                Japanese (9)
-              </option>
-              <option
-                value='portugese'>
-                Portugese (6)
-              </option>
-            </select>
-          </div>
+        <hr
+          className='belowHeaderLine'>
+        </hr>
+        <div>
+          <ReviewsListEntry
+            reviews={reviews}
+          />
         </div>
       </div>
-      <div>
-        <ReviewsListEntry
-          reviews={reviews}
-        />
+      <div
+        className='footer'>
+        {<img
+          className='footerSiteMapIcon'
+          src={reviews[0].collateral.footerSiteMapIcon}>
```

Comment:
If i understand correctly this is the footer for the whole page. This shouldn't be here when we put it all together it becomes more difficult to adjust